packages/web/localizations/fa.json

```diff
@@ -175,7 +175,13 @@
       "SORTMobile": "مرتب سازی"
     },
     "table": {
-      "poolId": "استخر #{id}"
+      "poolId": "استخر #{id}",
+      "stable": "مبادله پایدار",
+      "weighted": "وزن دار",
+      "internal": "مشوق های داخلی",
+      "external": "مشوق های خارجی",
+      "superfluid": "ابر سیال",
+      "noIncentives": "بدون انگیزه"
```

Comment:
@jonator just double checking if this was your intention. by committing the machine translations, contributors have a lower incentive to contribute because they don't see that translations are missing. 

From https://github.com/osmosis-labs/osmosis-frontend/pull/1575#issuecomment-1554687897

> I would definitely refrain from committing machine translations to the repo. In other words, use the translate command right before the build step ensuring that Osmosis is fully translated but also ensuring that contributors see that translations are missing and that they can contribute. Otherwise, you are running into the risk of having a translated osmosis front-end with bad machine translations.

Reply:
Yeah I'd rather have translations be present and not be perfect, then for them to be missing and the user would see the JSON key path instead.

Reply:
> Yeah I'd rather have translations be present and not be perfect, then for them to be missing and the user would see the JSON key path instead. 

If you run machine translate right before the build step (in the build script), users would never face missing translations.

Reply:
IMO it should be run separately as a separation of concerns. I'm not against adding pre build steps, but it should be necessary for _every_ build, and should semantically be "building" the app.

It's for the same reasons we don't run tests before every build. Though I _would_ be a fan of running an `inlang test translations` command to do a check of translations to see if it's necessary to run `yarn translate`.